playbooks/labkeeper-allinone-deployment-test/run.yaml

```diff
@@ -0,0 +1,37 @@
+- hosts: all
+  tasks:
+    - name: install required packages
+      become: true
+      shell: |
+        apt update -y
+        apt install python python-pip python3 python3-pip kpartx qemu-utils curl python-yaml \
+            debootstrap libffi-dev libssl-dev -y
+        pip install ansible
+      args:
+        executable: /bin/bash
+
+    - name: Prepare ubuntu user for running labkeeper
+      become: yes
+      no_log: yes
+      shell: |
+        useradd -m -d /home/ubuntu -s /bin/bash ubuntu
+        echo ubuntu:ubuntu | chpasswd
+        su - ubuntu -c "
+            ssh-keygen -f /home/ubuntu/.ssh/id_rsa -t rsa -N ''
+            cat /home/ubuntu/.ssh/id_rsa.pub > /home/ubuntu/.ssh/authorized_keys
+        "
+      args:
+        executable: /bin/bash
+
+    - name: Run labkeeper functional test
+      shell: |
+        sudo cp /home/ubuntu/.ssh/id_rsa ./openlab.pem
```

Comment:
Does these sudo is necessary? I guess not.

Reply:
labkeeper need the openlab.pem, you can see the `ansible.cfg` in labkeeper source code. hah

Reply:
I mean these `sudo` : ). After F2F discussion, we can set
```
become: true
```
Good to learn it!